content/zh/docs/tasks/administer-cluster/manage-resources/cpu-constraint-namespace.md

```diff
@@ -13,25 +15,36 @@ weight: 40
 <!-- overview -->
 
 <!--
-This page shows how to set minimum and maximum values for the CPU resources used by Containers
-and Pods in a namespace. You specify minimum and maximum CPU values in a
-[LimitRange](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#limitrange-v1-core)
+This page shows how to set minimum and maximum values for the CPU resources used by containers
+and Pods in a {{< glossary_tooltip text="namespace" term_id="namespace" >}}. You specify minimum
+and maximum CPU values in a
+[LimitRange](/docs/reference/kubernetes-api/policy-resources/limit-range-v1/)
 object. If a Pod does not meet the constraints imposed by the LimitRange, it cannot be created
 in the namespace.
 -->
-本页介绍如何为命名空间中容器和 Pod 使用的 CPU 资源设置最小和最大值。
+ 本页介绍如何设置{{< glossary_tooltip text="命名空间" term_id="namespace" >}}中的容器和 Pod 使用的 CPU 资源的最小和最大值。
```

Comment:
nit: Leading space here.

Reply:
add space before
<img width="166" alt="image" src="https://user-images.githubusercontent.com/16032654/168942040-91d54527-6a30-494c-8dea-8fc6e7ae3ad0.png">

add space after
<img width="305" alt="image" src="https://user-images.githubusercontent.com/16032654/168941976-ad0e2598-188b-4fc9-9cdf-eeb21f62a933.png">

so I think the spaces shouldn't be added here. 😄

Reply:
Oh, I mean this space. 😜

```diff
-本页介绍如何为
+ 本页介绍如何为
```

Reply:
Just checked the diff of PR, it should have been fixed.  😃